tests/test_driver/firebase_storage/reference_e2e.dart

```diff
@@ -127,17 +127,23 @@ void setupReferenceTests() {
     });
 
     group('getDownloadURL', () {
-      test('gets a download url', () async {
-        Reference ref = storage.ref('flutter-tests/ok.txt');
-        await ref.putString('ok');
-
-        String downloadUrl = await ref.getDownloadURL();
-        expect(downloadUrl, isA<String>());
-        expect(downloadUrl, contains('ok.txt'));
-        expect(downloadUrl, contains(storage.app.options.projectId));
-      });
+      test(
+        'gets a download url',
+        () async {
+          Reference ref = storage.ref('flutter-tests/ok.txt');
+          await ref.putString('ok');
+
+          String downloadUrl = await ref.getDownloadURL();
+          expect(downloadUrl, isA<String>());
+          expect(downloadUrl, contains('ok.txt'));
+          expect(downloadUrl, contains(storage.app.options.projectId));
+        },
+        // Fails on emulator since iOS SDK 10. See PR notes:
+        // https://github.com/firebase/flutterfire/pull/9708
+        skip: defaultTargetPlatform == TargetPlatform.iOS ||
```

Comment:
For tracking - this will resolve with https://github.com/firebase/firebase-ios-sdk/pull/10370 (or perhaps something in firebase-tools)